components/item/SummaryInfo.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="card mb-3 ">
+    <header
+      v-if="titles"
+      class="card-heading px-4 pt-4"
```

Comment:
I think the "card-heading" class is no longer needed in this new structure. The class currently helps to organise title(s), image and description in the card.